datahub-classify/src/datahub_classify/infotype_helper.py

```diff
@@ -40,32 +41,11 @@
     nlp_english = spacy.load(spacy_model_name)
 spacy_models_list = [nlp_english]
 
-# TODO: pull out the common/repetitive code
 
-
-def inspect_for_email_address(metadata, values, config):
+def compute_name_description_dtype_score(
+    metadata: Metadata, config: dict, debug_info: dict
+) -> dict:
```

Comment:
it'd be good to start using dataclasses or TypedDict instead of raw `dict` types here and elsewhere in the code

Reply:
Not quite what I was looking for. Where possible, please use dataclasses. For example, debug_info shouldn't be a dictionary but instead an instance of a dataclass

Reply:
We have changed debug_info to be of TypedDict. Does this work?